Casks/t/tev.rb

```diff
@@ -2,21 +2,19 @@
   arch intel: "-intel"
 
   version "2.3.2"
-  sha256 arm:   "3d506f5b99d9dafc65d7bf784d3bcb0c4eab2440f4ace4089c47a7f22330cf1a",
-         intel: "392ad239912388f04b3521b91d7979148e3d11d9f09f4ae9a372f92e682670ee"
+  sha256 arm:   "3aee171b17e74c2bfe6ceb4bc504c2582dae58fc475dc6f8c4e6f140d3aa9a71",
+         intel: "4ca9bc5b8ce158668e32a6081f2290690cfd3b2d6716515bc235be09c1e911e6"
 
   url "https://github.com/Tom94/tev/releases/download/v#{version}/tev#{arch}.dmg"
   name "tev"
-  desc "HDR image comparison tool with an emphasis on OpenEXR images"
+  desc "High dynamic range (HDR) image viewer for people who care about colors"
```

Comment:
```suggestion
  desc "High dynamic range (HDR) image viewer"
```

According to our guidelines for `desc` stanzas, we like to keep these succinct, and without marketing notes.

Reply:
Does “High dynamic range image viewer with accurate color management” work as well? Actually handling colors correctly is an important distinguishing part compared to most other tools and I’d prefer to not drop it entirely.